### DISCUSSION TITLE
Rsbuild是否支持vue2.6

### DISCUSSION THREAD

**Discussion by ssccchh** [web-infra-dev/rsbuild#2015] [Q&A]
尝试了一下修改rspack的vue-loader，无法正常解析template，rsbuild是否无法通过配置的修改做到支持vue2.6
![image](https://github.com/web-infra-dev/rsbuild/assets/26810503/b005379e-e0a8-4409-9b99-8f6d46f7d3c2)

**Comment by chenjiahan** [comment]
![image](https://github.com/web-infra-dev/rsbuild/assets/7237365/1860d057-9877-4651-9a66-3f4fc5dbd324)

You can try this plugin: https://github.com/skymoonya/rsbuild-plugin-vue-legacy
